File: Lógica/Lista1/ExPySalaC/ex3.c
#include <stdio.h>
#include <math.h>

int main()
{
    float altura,raio,volume;

    printf("Digite a altura da lata de óleo ( em M ): ");
    scanf("%f",&altura);
    
    printf("Digite o raio da lata de óleo (em M): ");
    scanf("%f",&raio);
    
    volume = M_PI * (pow(raio,2)) * altura
    
    printf("Volume da lata de óleo: %.2f ",volume);
 
    return 0;
}

File: Lógica/Lista1/ExPySalaPython/ex3.py
import math
altura = float(input("Digite a altura da lata de óleo ( em M ): "))
raio = float(input("Digite o raio da lata de óleo (em M) : "))
volume = math.pi * (raio **2) * altura
print(f"Volume da lata de óleo: {volume:.2f}")